README.md

```diff
@@ -57,26 +62,28 @@ composer create-project --keep-vcs ezsystems/ezplatform-ee ezplatform ^2
 cd ezplatform
 ```
 
-_Note: If composer is installed locally instead of globally, the first command will start with `php composer.phar`._
+**Note:** If composer is installed locally instead of globally, the first command will start with `php composer.phar`.
+
+During the installation process you will be asked to provide database host name, login, password, etc.
+The configuration details will be placed in `<ezplatform>/app/config/parameters.yml`.
 
-During the installation process you will be asked to input things like database host name, login, password, etc.
-They will be placed in `<ezplatform>/app/config/parameters.yml`.
+Next, you will receive instructions on how to install data into the database, and how to run a simplified dev server using the `bin/console server:run` command.
 
-Next you will receive instructions on how to install data into the database, and how to run a simplified dev server using the `bin/console server:run` command.
-_Tip: For a more complete and better performing setup using Apache or Nginx, read up on how to [install eZ Platform manually](https://doc.ezplatform.com/en/latest/getting_started/install_manually/)._
+**Tip:** For a more complete and better performing setup using Apache or Nginx, see how to [install eZ Platform manually](https://doc.ezplatform.com/en/latest/getting_started/install_manually/).
 
 ## Issue tracker
-Submitting bugs, improvements and stories is possible on https://jira.ez.no/browse/EZEE.
-If you discover a security issue, please see how to responsibly report such issues on https://doc.ez.no/Security.
+Submitting bugs, improvements and stories is possible on [https://jira.ez.no/browse/EZEE](https://jira.ez.no/browse/EZEE).
+If you discover a security issue, please see how to responsibly report such issues in ["Reporting security issues in eZ Systems products"](https://doc.ezplatform.com/en/latest/guide/reporting_issues/#reporting-security-issues-in-ez-systems-products).
 
 ## Backwards compatibility
-eZ Platform aims to be **100% content compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning that content in those versions of the CMS can be upgraded using
+eZ Platform aims to be **fully content compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning that the content in those versions of the CMS can be upgraded using
```

Comment:
Question: does the upgrade procedure actually covers anything below eZ Publish Platform 5.4?

Reply:
As for me, it sounds to be a bit hasty... For our own safety I would prefer to stay with:

```suggestion
eZ Platform aims to be **fully content compatible** with eZ Publish 5.x, meaning that the content in this versions of the CMS can be upgraded using
```

_Note_: We can instead mention somewhere that it **should** be possible to smoothly upgrade both eZ Publish 3.x and 4.x to 5.4 before upgrading to eZ Platform.